sig-ui/annual-report-2021.md

```diff
@@ -0,0 +1,102 @@
+# SIG UI 2021 Annual Report
+
+**Authors:**
+
+Sebastian Florek ([@floreks](https://github.com/floreks))
+Marcin Maciaszczyk ([@maciaszczykm](https://github.com/maciaszczykm))
+Jeffrey Sica ([@jeefy](https://github.com/jeefy))
+
+- [Operational](#operational)
+- [Membership](#membership)
+- [Current initiatives and project health](#current-initiatives-and-project-health)
+
+## Operational
+
+**How are you doing with operational tasks in [sig-governance.md](/committee-steering/governance/sig-governance.md)?**
+
+The SIG primarily works asynchronously. In the past, our meetings would last less than five minutes and be relatively disruptive. Since the release of the 2.0 Dashboard, meeting attendance transitioned from optional to near-non-existent. Instead, we coordinate through the #sig-ui Slack channel and handle any project planning via GitHub. All chairs are regular contributors of the Dashboard subproject and help review PRs and triage issues. 
+
+**Is your README accurate? have a CONTRIBUTING.md file?**
+
+Yes, our README is accurate and is typically updated with every new Dashboard release. Also we do have a CONTRIBUTING.md file however it has not been updated with any more info beyond the standard template.
```

Comment:
I'd call out that this is in the dashboard repo and not k/community. The question itself was specifically referencing the files as part of your sig folder here in k/community

Reply:
Yeah, @jeefy can we mention here the https://github.com/kubernetes/community/tree/master/sig-ui? It probably needs a little adjustment as the meetings are not bi-weekly anymore.